CHANGELOG.md

```diff
@@ -8,7 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Support for Wang sets.
-- Support for Tiled 1.9+ `class` property.
+- Support for Tiled 1.9 `Class` property. Maps, tilesets and layers now have a `user_type` property. 
+
+### Deprecated
+- `Tile::tile_type`: Use `Tile::user_type` instead.
+- `Object::obj_type` Use `Object::user_type` instead.
```

Comment:
When something is "Deprecated" I expect it to still be available, but as far as I understand the patch, these have been "Removed"?

Reply:
They're still here, but marked as deprecated. I renamed them at first, but then decided to keep old names as "deprecated":
<img width="609" alt="Screenshot 2022-12-16 at 11 08 41" src="https://user-images.githubusercontent.com/278340/208075058-002453ee-4f6c-4efe-9185-0f324d6cb211.png">

Reply:
You're right, I must have misinterpreted the diff, sorry.